packages/cardano-graphql/package.json

```diff
@@ -36,6 +36,7 @@
     "@graphql-codegen/typescript-graphql-request": "~4.3.4",
     "@graphql-codegen/typescript-operations": "~2.2.4",
     "@graphql-tools/utils": "~8.6.1",
+    "@types/validator": "^13.7.1",
```

Comment:
Seems like this was added in the wrong package?

Reply:
Actually, I got multiple `validator`errors about missing package while building the app. That is the reason I added / fixed this.

Reply:
Best to then commit this separately as a `fix(cardano-graphql)`

Reply:
@rhyslbw I checked this one more time and without `@types/validator` I'm getting this:
![Screenshot 2022-03-14 at 19 37 54](https://user-images.githubusercontent.com/18653950/158239234-ccea66d6-e4f0-4fd9-bc59-463ef8df94c6.png)